### ISSUE TITLE
EntityType has wrong docs for preferred_choice

### ISSUE THREAD

**Issue by ChangePlaces** [symfony/symfony-docs#10034]
It seems to be using the defacto choicetype docs rather than detailing how to select certain entities for the preferred choices.

https://symfony.com/doc/current/reference/forms/types/entity.html

**Comment by jamm-kevin** [comment]
Furthermore it seems that preferred_choice does not work the way it is documented. I need it and i do not find the trick to make it work.

**Comment by ChangePlaces** [comment]
@jamm-kevin Unfortunately you can't pass in id's of the entities that you want (you could with symfony 3 I believe). Instead you have to pass in the actual entity, e.g. (untested) `'preferred_choices' => [$repo->find(1), $repo->find(2)]`

**Comment by yceruto** [comment]
Agree that this example doesn't fit to this form type:
```php
use Symfony\Component\Form\Extension\Core\Type\ChoiceType;
// ...

$builder->add('language', ChoiceType::class, array(
    'choices' => array(
        'English' => 'en',
        'Spanish' => 'es',
        'Bork'   => 'muppets',
        'Pirate' => 'arr',
    ),
    'preferred_choices' => array('muppets', 'arr'),
));
```
Instead, it should be something like this:
```php
use App\Entity\User;
use Symfony\Bridge\Doctrine\Form\Type\EntityType;
// ...

$builder->add('users', EntityType::class, array(
    'class' => User::class,
    'preferred_choices' => $group->getPreferredUsers(),
));
```
and perhaps explain in this case that it must pass the instances of the users.

**Comment by jamm-kevin** [comment]
@ChangePlaces Thanks a lot ; passing entire instances of my entities does the trick.
Definitely a documentation issue for me. I would never EVER had the idea to pass entire entities since the doc pass ids or string values.

**Comment by ChangePlaces** [comment]
@javiereguiluz I think the above can be put in the documentation with a brief mention. Maybe there are other ways of showing the preferred choices that I'm not aware of.

**Comment by javiereguiluz** [comment]
Fixed by #10072.
